README.md

```diff
@@ -6,9 +6,35 @@ Codidact is community-driven Q&A software: created, maintained, used, and run by
 money-making company behind this.
 
 We're currently in the early stages of development, but once we have a working MVP, we'll be hosting a public Codidact instance
-for everyone to use. The software will also be available to download and run yourself to host your own instance under your own
+for everyone to use. The software will also be available to download and run yourself to host your instance under your own
 rules.
 
+## Installation
+These instructions are for setting up and running a local instance of Codidact.
+
+#### Windows
+We'll be using Visual Studio for this setup.
+
+1. [Download Visual Studio 2019 Community](https://visualstudio.microsoft.com/downloads/) and start up the installer. If you
+   already have VS2019 installed, run the Visual Studio Installer instead.
+2. When prompted to select workloads, select the "ASP.NET and Web Development" workload, and then install it.
+3. Once the installation is complete, clone/download this repository and extract it somewhere.
+4. In the root of the repository, open the `Codidact.sln` file with visual studio to get started.
+5. Once Visual Studio has finished loading, you can run Codidact by setting the WebUI project as the startup project.
+
+Alternatively, if you don't want to run Visual Studio just to start the project, you can navigate to the src/WebUI folder
+(so that `WebUI.csproj` is in your working directory) and issue the command `dotnet run`.
```

Comment:
Maybe we should add a notice, that VS automatically installs all missing packages and a link to this resource, that explains what to do if this doesn't work:

https://docs.microsoft.com/en-US/nuget/consume-packages/package-restore

(At least I needed it for my installation)